hack/lib/catalogsource.bash

```diff
@@ -73,11 +68,10 @@ spec:
         - /bin/sh
         - -c
         - |-
-          podman login -u $pull_user -p $token --tls-verify=false image-registry.openshift-image-registry.svc:5000
-          mkdir -p /database && \
-          /bin/opm registry add                         -d /database/index.db --mode=replaces -b docker.io/markusthoemmes/serverless-bundle:1.7.2
-          /bin/opm registry add --container-tool=podman -d /database/index.db --mode=replaces -b image-registry.openshift-image-registry.svc:5000/$OLM_NAMESPACE/serverless-bundle && \
-          /bin/opm registry serve -d /database/index.db -p 50051
+          podman login -u $pull_user -p $token image-registry.openshift-image-registry.svc:5000 && \
+          /bin/opm registry add                         -d index.db --mode=replaces -b docker.io/markusthoemmes/serverless-bundle:1.7.2 && \
```

Comment:
`docker.io/markusthoemmes/serverless-bundle` this is to be changed in different PR?

Reply:
No that's a completely separate image. That'll stay there I assume for 1.8.0.

Reply:
I know that other image. I'm asking should that be moved to openshift-knative org on quay.io as well?